src/Readme.md

```diff
@@ -187,4 +187,24 @@ Output Example executing the SCANBUILD analysis with `winagent` target:
 [CleanInternals: PASSED]
 [ScanBuild: PASSED]
 <winagent>[SCANBUILD: PASSED]<winagent>
-```
\ No newline at end of file
+```
+
+## Compile Wazuh on Alpine Linux
```

Comment:
Regarding the whole content of this readme, I think it has no sense to include here the guide to installing the agent in Alpine Linux. Let's remove this change and let the documentation explain that.

Reply:
OK, I delete this section !